lib/browser/desktop-capturer.ts

```diff
@@ -80,3 +80,11 @@ export const getSourcesImpl = (event: Electron.IpcMainEvent | null, args: Electr
 
   return getSources;
 };
+
+export const setSkipCursorImpl = (event: Electron.IpcMainEvent | null, sourceId: string, skipCursor: boolean) => {
+  const capturer: ElectronInternal.DesktopCapturer | null = createDesktopCapturer();
```

Comment:
This is a memory leak. `createDesktopCapturer()` returns an object which is not released until capturing has finished.

Reply:
Not creating a desktopcapturer anymore. calling the static method directly